dev/io.openliberty.mcp.internal/src/io/openliberty/mcp/internal/McpCdiExtension.java

```diff
@@ -37,13 +39,53 @@ void registerTools(@Observes ProcessManagedBean<?> pmb) {
         for (AnnotatedMethod<?> m : type.getMethods()) {
             Tool toolAnnotation = m.getAnnotation(Tool.class);
             if (toolAnnotation != null) {
-                registerTool(toolAnnotation, pmb.getBean(), m, javaClass.getName());
+                String qualifiedName = javaClass.getName() + "." + m.getJavaMember().getName() + "()";
+                registerTool(toolAnnotation, pmb.getBean(), m, qualifiedName);
```

Comment:
```suggestion
                String qualifiedName = javaClass.getName() + "." + m.getJavaMember().getName();
                registerTool(toolAnnotation, pmb.getBean(), m, qualifiedName);
```

Don't add brackets like this when referring to a method.

Either include correct information about method parameters, or just use the name.